app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>All Orders</h1>
+
+<% @orders.each do |order|%>
+  <tr>
+  <section id = "order_<%=order.id%>">
+      <td><p><%=link_to order.user.name, "/admin/profile/#{order.user.id}"%></p></td>
+      <td><p><%= "Order Number: #{order.id}" %></p></td>
+      <td><p><%= "Date Created: #{order.created_at.strftime("%m/%d/%Y")}" %></p></td>
```

Comment:
Oh nice. We didn't use strftime and kept created_at as is. However, this is much better for the user